ld/devices.data

```diff
@@ -573,7 +573,7 @@ stm32l1 END ROM_OFF=0x08000000 RAM_OFF=0x20000000 CPU=cortex-m3 FPU=soft
 stm32l4 END ROM_OFF=0x08000000 RAM_OFF=0x20000000 RAM2_OFF=0x10000000 RAM3_OFF=0x20040000 CPU=cortex-m4 FPU=hard-fpv4-sp-d16
 stm32g0 END ROM_OFF=0x08000000 RAM_OFF=0x20000000 CPU=cortex-m0plus FPU=soft
 stm32g4 END ROM_OFF=0x08000000 RAM_OFF=0x20000000 CPU=cortex-m4 FPU=hard-fpv4-sp-d16
-stm32h7 END ROM_OFF=0x08000000 ROM1_OFF=0x08100000 RAM_OFF=0x24000000 RAM1_OFF=0x30000000 RAM2_OFF=0x30020000 RAM3_OFF=0x30040000 RAM4_OFF=0x38000000 CCM_OFF=0x20000000 CPU=cortex-m7 FPU=hard-fpv5-dp-d16
+stm32h7 END ROM_OFF=0x08000000 ROM1_OFF=0x08100000 RAM_OFF=0x24000000 RAM1_OFF=0x30000000 RAM2_OFF=0x30020000 RAM3_OFF=0x30040000 RAM4_OFF=0x38000000 CCM_OFF=0x20000000 CPU=cortex-m7 FPU=hard-fpv5-d16
```

Comment:
This looks like the same problem as f7.  They have started selling both single and double precisions H7s...
![Screenshot from 2022-02-25 22-14-53](https://user-images.githubusercontent.com/127907/155810369-44878524-12dc-4e14-a191-f21fc1ce1185.png)
Your "fix" will just break it for other people.  You'll have to split it into two targets, or otherwise come up with a fancy idea.  (This is still an unsolved issue for f7)

Reply:
Based on PM0253, Table 14. all Cortex-M7 cores in STM32H7 are double-precision and single-precision capable. This is different than F7, where certain cores are SP-only and others are SP-and-DP. The SP-FPU and DP-FPU probably reflects fact that certain dual-core configurations have DP+SP Cortex-M7 and SP-only Cortex-M4.